src/pam/oslogin_sshca.cc

```diff
@@ -277,27 +278,25 @@ static size_t ExtractFingerPrint(const char *extension, char **out) {
   return i;
 }
 
-static int GetByoidFingerPrint(pam_handle_t *pamh, const char *blob, char **fingerprint) {
+static int GetByoidFingerPrint(const char *blob, char **fingerprint) {
   size_t f_len, k_len, exts_len = -1;
   char *key, *exts = NULL;
 
   k_len = SplitKey(blob, &key);
   if (k_len <= 0) {
-    PAM_SYSLOG(pamh, LOG_ERR, "Could not split ssh ca cert.");
+    SysLogErr("Could not split ssh ca cert.");
```

Comment:
suggestion: "SSH CA" instead of "ssh ca"

Reply:
updated, thanks.